gh-98239: Document that `inspect.getsource()` can raise `TypeError` (#101689)
python/cpython: Doc/library/inspect.rst

   object.  The source code is returned as a list of the lines corresponding to the
   object and the line number indicates where in the original source file the first
   line of code was found.  An :exc:`OSError` is raised if the source code cannot

>    be retrieved.  This
>    will fail with a :exc:`TypeError` if the object is a built-in module, class, or
>    function.

Reviewer: ```suggestion
   be retrieved.
   A :exc:`TypeError` is raised if the object is a built-in module, class, or
   function.
```

   Return the text of the source code for an object. The argument may be a module,
   class, method, function, traceback, frame, or code object.  The source code is
   returned as a single string.  An :exc:`OSError` is raised if the source code

>    cannot be retrieved.  This
>    will fail with a :exc:`TypeError` if the object is a built-in module, class, or
>    function.

Reviewer: ```suggestion
   cannot be retrieved.
   A :exc:`TypeError` is raised if the object is a built-in module, class, or
   function.
```
